.github/workflows/review-apps.yml

```diff
@@ -0,0 +1,130 @@
+# See https://developer.github.com/v3/
+# and https://help.github.com/en/actions
+name: 🕵 Review apps
+
+# Run this pipeline when a label is added and when a push is made on this PR.
+on:
+  pull_request:
+    branches: [ master ]
+    types: [ labeled, synchronize ]
+
+env:
```

Comment:
Du coup on peut pas mutualiser les variables d'environnement pour différents workflow j'ai l'impression… J'imagine qu'on peut pas faire un `cat clevercloud.env` par exemple… Et ça marcherait pour nous le [`.clever.json`](https://github.com/47ng/actions-clever-cloud#specifying-the-application-to-deploy) ?

Reply:
yaml permettent les inclusions (anchors) mais ça a pas l'air d’être le cas pour les actions.
Update: github ne respecte pas la norme yaml pour les actions, c'est tout pourri

Reply:
Hum...
Le fichier `clever.json` est créé puis alimenté automatiquement quand on utilise le CLI de Clever. Il ne contient aucune variable d'environnement et ne permet pas d'en charger.
@ikarius, c'est exactement ce dont on aurait besoin... (soupir)

Reply:
je vais faire une passe dessus...